azure-webapp-gradle-plugin/src/main/java/lenala/azure/gradle/webapp/AzureWebAppExtension.java

```diff
@@ -3,6 +3,7 @@
 import lenala.azure.gradle.webapp.configuration.AppService;
 import lenala.azure.gradle.webapp.configuration.Authentication;
 import lenala.azure.gradle.webapp.configuration.Deployment;
+import lenala.azure.gradle.webapp.configuration.DeploymentSlotDeployTarget;
```

Comment:
This import looks unused?

Reply:
Yep, might have forgotten about that! I'm on vacation and will be able to change it next week.